test/jobs/validate_dnssec_job_test.rb

```diff
@@ -68,4 +68,77 @@ def test_job_should_not_set_validation_datetime_if_validation_is_invalid
     @domain.reload
     assert_nil @domain.dnskeys.first.validation_datetime
   end
+
+  def test_prepare_validator_configures_dnsruby_resolver_with_correct_parameters
+    job = ValidateDnssecJob.new
+    
+    # To store original environment variable
+    original_timeout = ENV['nameserver_validation_timeout']
+    
+    # Seadista kindel väärtus testiks
+    ENV['nameserver_validation_timeout'] = '4'
```

Comment:
Translate comment to English, because registry is Open Source product.
Also, among the English comments, this one stands out.